src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -80,7 +80,6 @@
     private Submission submission;
 
     @OneToMany(mappedBy = "result", cascade = CascadeType.ALL, orphanRemoval = true)
-    @OrderColumn
```

Comment:
I think this annotation is important for the database, you cannot simply remove it.